website/project/model.py

```diff
@@ -1638,7 +1638,7 @@ def use_as_template(self, auth, changes=None, top_level=True):
         new.nodes = [
             x.use_as_template(auth, changes, top_level=False)
             for x in self.nodes
-            if x.can_view(auth)
+            if x.can_view(auth) and not x.is_deleted
```

Comment:
I also think not. Our code will need to distinguish between an auth error vs. a deleted error (e.g. 403 vs 410).